utils/rpms/daos.spec

```diff
@@ -552,6 +554,9 @@ getent passwd daos_agent >/dev/null || useradd -s /sbin/nologin -r -g daos_agent
 # No files in a shim package
 
 %changelog
+* Fri Mar 17 2023 Tom Nabarro <tom.nabarro@intel.com> 2.3.106-2
+- Add numactl requires for server and agent
```

Comment:
Looks like the ticket was updated to just prescribe this for `daos_admin`.

Reply:
done